pkg/scraper/scraper.go

```diff
@@ -38,63 +37,40 @@ const (
 )
 
 var (
-	summaryRequestLatency = metrics.NewHistogramVec(
+	requestDuration = metrics.NewHistogramVec(
 		&metrics.HistogramOpts{
 			Namespace: "metrics_server",
-			Subsystem: "kubelet_summary",
+			Subsystem: "kubelet",
 			Name:      "request_duration_seconds",
-			Help:      "The Kubelet summary request latencies in seconds.",
-			// TODO(directxman12): it would be nice to calculate these buckets off of scrape duration,
```

Comment:
Is this TODO outdated?

Reply:
Yes, I don't think it makes sense to introduce dynamic buckets.